astropy/units/physical/__init__.py

```diff
@@ -0,0 +1,49 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+"""Defines the physical types that correspond to different units.
+
+The classes and functions defined here are also available in
+(and should be used through) the `astropy.units` namespace.
+"""
+
+from . import core
+from .core import *
+from .core import _name_physical_mapping  # sphinx-astropy needs this when building docs
```

Comment:
The comment is confusing: it is not sphinx that needs it, sphinx just uses the `__doc__` that is created here in its html pages. Maybe just `#  Needed to create __doc__ below`

Reply:
When I wrote that `sphinx-astropy` needs this, I meant that `sphinx-astropy` needs this: https://github.com/astropy/sphinx-astropy/blob/9ee5275a8ccc09f434dc54f66993dbd0356d7200/sphinx_astropy/conf/v1.py#L149

Reply:
Oops, so much for my reading skills... And great that you caught that!

Reply:
`sphinx_astropy` has `conf/v1.py` and `conf/v2.py`. I have already shown that `v1.py` accesses `physical._name_physical_mapping`, but I should also point out the analogous line in `v2.py`: https://github.com/astropy/sphinx-astropy/blob/9ee5275a8ccc09f434dc54f66993dbd0356d7200/sphinx_astropy/conf/v2.py#L140

Reply:
The fact that this is in `sphinx-astropy` still seems weird - shouldn't astropy itself just export those references? Anyway, not something this PR has to address, though I think it would be good to expand the comment.